amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -711,4 +717,7 @@ type ColonyAction @model {
   pendingDomainMetadataId: ID
   pendingDomainMetadata: DomainMetadata
     @hasOne(fields: ["pendingDomainMetadataId"])
+  pendingColonyMetadataId: ID
+  pendingColonyMetadata: ColonyMetadata
+    @hasOne(fields: ["pendingColonyMetadataId"])
```

Comment:
Will this not lead to the same problem as we're currently facing with pending domain metadata, where the values can get overwritten?